neqo-transport/src/ecn.rs

```diff
@@ -313,11 +318,6 @@ impl Info {
             .count()
             .try_into()
             .expect("usize fits into u64");
-        if newly_acked_sent_with_ect0 == 0 {
-            qwarn!("ECN validation failed, no ECT(0) packets were newly acked");
-            self.disable_ecn(stats, ValidationError::Bleaching);
-            return;
-        }
```

Comment:
Say the packets acked here were sent during `ValidationState::NotStarted` or `ValidationState::Unknown`, they would thus not be marked, as expected. This is not a reason to disable ECN. 

This would fix https://github.com/mozilla/neqo/issues/2741.

@larseggert please double check the above reasoning.

Reply:
The logic is "if we sent any of the newly acked packets with ECN marks, but the count didn't increase in the ACK, disable".

Reply:
> if we sent any of the newly acked packets with ECN marks, but the count didn't increase in the ACK, disable

@larseggert that check is happening further below. I am not intending to delete it.

https://github.com/mozilla/neqo/blob/531cb0b6c21d27ff1ccec5502ecc346a6a922d71/neqo-transport/src/ecn.rs#L323-L327

Reply:
With the thumbps-up emoji, I am considering this resolved.